src/r0/directory/set_room_visibility.rs

```diff
@@ -17,8 +18,7 @@ ruma_api! {
     request {
         /// The ID of the room of which to set the visibility.
         #[ruma_api(path)]
-        pub room_id: String,
-
+        pub room_id: RoomId,
```

Comment:
I added empty lines between all documented request / response fields recently because they often also have attributes, making them three lines or longer, and I feel then the extra empty lines actually improve readability (for single-line fields, they rather hinder it IMHO).

```suggestion
        pub room_id: RoomId,
        
```

Reply:
:cursing_face:

```suggestion
        pub room_id: RoomId,

```